### ISSUE TITLE
[Bug]: Exception Error / Player Tick Event

### ISSUE THREAD

**Issue by jilienecircelli** [Zepalesque/Aether-Redux-Issue-Tracker#51]
### Minecraft Version

1.19.2

### Mod Version

1.3.3

### Forge/Neoforge Version

43.3.5

### What happened?

When entering the Aether and walking around, player gets kicked and then can't rejoin the game if they are still in the Aether. Currently teleporting players out of the dimension to let them back in.

Error in Console:
```
[Server thread/ERROR] [net.minecraftforge.eventbus.EventBus/EVENTBUS]: Exception caught during firing event: class net.zepalesque.redux.world.feature.config.TestBelowPlaceConfig cannot be cast to class net.minecraft.world.level.levelgen.feature.configurations.RandomPatchConfiguration (net.zepalesque.redux.world.feature.config.TestBelowPlaceConfig is in module aether_redux@1.19.2 of loader 'TRANSFORMER' @5e1a7d3; net.minecraft.world.level.levelgen.feature.configurations.RandomPatchConfiguration is in module minecraft@1.19.2 of loader 'TRANSFORMER' @5e1a7d3)
	Index: 8
	Listeners:
		0: HIGH
		1: ASM: class io.github.edwinmindcraft.calio.common.CalioEventHandler playerFirstTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		2: ASM: class dev.architectury.event.forge.EventHandlerImplCommon event(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		3: NORMAL
		4: ASM: xaero.pac.common.event.CommonEventsForge@10796c72 onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		5: ASM: class net.mcreator.ulterlands.procedures.GrantDiaScienceProcedure onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		6: ASM: class net.mcreator.ulterlands.procedures.GrantCoverStarsProcedure onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		7: ASM: class io.github.edwinmindcraft.origins.common.OriginsEventHandler onPlayerTickEnd(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		8: ASM: class org.zeith.improvableskills.data.PlayerDataManager playerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		9: net.minecraftforge.eventbus.EventBus$$Lambda$1573/0x00007f15fc7006b8@73382525
		10: ASM: class net.mcreator.unusualend.procedures.TPConfirmProcedure onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		11: ASM: class net.mcreator.unusualend.procedures.ReturnMusicProcedure onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		12: ASM: class net.mcreator.unusualend.procedures.PlayerTickUpdateShieldProcedure onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		13: ASM: class net.mcreator.unusualend.procedures.PlayerTickUpdateMusicProcedure onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		14: ASM: class net.mcreator.unusualend.procedures.PlayerTickPlayMusicProcedure onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		15: ASM: class net.mcreator.unusualend.procedures.PlayerTickOverlayCountProcedure onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		16: ASM: class net.mcreator.unusualend.procedures.HoustonWeHaveAProblemProcedure onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		17: ASM: class net.thirdlife.iterrpg.procedures.MeleeCooldownProcedure onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		18: ASM: class net.thirdlife.iterrpg.procedures.ManaCalculateProcedure onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		19: ASM: class net.thirdlife.iterrpg.procedures.ArmorCooldownProcedure onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		20: net.minecraftforge.eventbus.EventBus$$Lambda$1573/0x00007f15fc7006b8@c1fc32c
		21: ASM: class com.mrcrayfish.guns.common.ReloadTracker onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		22: ASM: xaero.common.events.FMLCommonEventHandler@778aba98 handlePlayerTickStart(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		23: ASM: class net.mcreator.phantasm.procedures.EnterUnderislandGrantProcedure onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		24: ASM: class com.momosoftworks.coldsweat.common.event.TempEffectsCommon onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		25: ASM: class com.momosoftworks.coldsweat.common.event.ArmorInsulation addLeatherModifiers(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		26: ASM: class com.momosoftworks.coldsweat.common.capability.EntityTempManager playerRiding(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		27: ASM: class com.momosoftworks.coldsweat.common.capability.EntityTempManager onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		28: ASM: class com.momosoftworks.coldsweat.common.capability.ItemInsulationManager onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		29: ASM: se.mickelus.tetra.effect.ItemEffectHandler@4e9dca15 onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		30: ASM: xaero.map.events.FMLCommonEvents@7b9590f8 handlePlayerTickStart(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		31: ASM: org.infernalstudios.shieldexp.events.ShieldExpansionEvents@3b2c0dbe onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		32: ASM: class com.aetherteam.aether.event.listeners.EntityListener onRiderTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		33: ASM: class com.aetherteam.aether.event.listeners.DimensionListener onPlayerTraveling(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		34: ASM: com.github.L_Ender.cataclysm.event.ServerEventHandler@dc77c18 onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		35: ASM: com.mrcrayfish.framework.event.ForgeEvents@4e924218 onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		36: net.minecraftforge.eventbus.EventBus$$Lambda$1573/0x00007f15fc7006b8@5ef32a59
		37: ASM: class com.teamabnormals.personality.core.other.PersonalityEvents onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		38: ASM: class net.mcreator.enlightened_end.procedures.RadioactiveItemsProcedure onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		39: ASM: class net.mcreator.enlightened_end.procedures.RadiationTickProcedure onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		40: ASM: class net.mcreator.enlightened_end.procedures.PlayerRadiationProcedure onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		41: ASM: class net.mcreator.enlightened_end.procedures.PermafrostCursePlayerProcedure onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		42: ASM: class net.mcreator.enlightened_end.procedures.OozeSeepsParticlesProcedure onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		43: ASM: class net.mcreator.enlightened_end.procedures.OozeRainParticlesProcedure onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		44: ASM: class net.mcreator.enlightened_end.procedures.FuschiaParticlesProcedure onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		45: ASM: class net.mcreator.enlightened_end.procedures.FreezingParticlesProcedure onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		46: ASM: class net.mcreator.enlightened_end.procedures.ExoGasParticlesProcedure onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		47: ASM: class net.mcreator.enlightened_end.procedures.EndMusicProcedure onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		48: ASM: class net.mcreator.enlightened_end.procedures.EndBarrenAmbienceProcedure onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		49: ASM: class net.mcreator.enlightened_end.procedures.DontLookDownTickProcedure onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		50: ASM: class net.mcreator.enlightened_end.procedures.ChorusParticlesProcedure onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		51: ASM: class net.mcreator.enlightened_end.procedures.CeruleanParticlesProcedure onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		52: ASM: class com.teamabnormals.upgrade_aquatic.core.other.UAEvents onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		53: ASM: class net.mcreator.endlessbiomes.procedures.VoidRisesProcedure onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		54: ASM: class net.mcreator.endlessbiomes.procedures.HexMilkProcedure onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		55: ASM: class net.mcreator.endlessbiomes.procedures.GiveHalfEffectBackProcedure onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		56: ASM: vazkii.quark.content.building.module.HollowLogsModule@f065034 playerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		57: ASM: vazkii.quark.content.management.module.AutomaticToolRestockModule@393e09cd onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		58: ASM: vazkii.quark.content.tools.module.PathfinderMapsModule@de00052 playerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		59: ASM: vazkii.quark.content.tools.module.ColorRunesModule@51b0179c onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		60: ASM: vazkii.quark.content.tweaks.module.CampfiresBoostElytraModule@7ebae343 onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		61: ASM: vazkii.quark.content.tweaks.module.EnhancedLaddersModule@173ef4ac onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		62: ASM: vazkii.quark.content.tweaks.module.GrabChickensModule@2ef147bd playerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		63: ASM: vazkii.quark.content.tweaks.module.CompassesWorkEverywhereModule@74c994cd onUpdate(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		64: ASM: class net.mehvahdjukaar.supplementaries.common.events.forge.ServerEventsForge onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		65: ASM: class net.mcreator.undead_unleashed.procedures.MetalheadTriggerProcedure onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		66: ASM: class com.stal111.forbidden_arcanus.event.TickListener onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		67: ASM: class com.stal111.forbidden_arcanus.common.event.PlayerEvents onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		68: ASM: class com.stal111.forbidden_arcanus.client.event.ClientPlayerEvents onClientPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		69: ASM: class com.simibubi.create.content.equipment.bell.HauntedBellPulser hauntedBellCreatesPulse(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		70: ASM: air onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		71: net.minecraftforge.eventbus.EventBus$$Lambda$1573/0x00007f15fc7006b8@7eac2365
		72: ASM: class com.momosoftworks.coldsweat.core.event.TaskScheduler runScheduledTasks(Lnet/minecraftforge/event/TickEvent;)V
		73: LOW
		74: ASM: class io.github.edwinmindcraft.calio.common.CalioEventHandler playerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		75: ASM: se.mickelus.tetra.items.forged.VibrationDebuffer@123948f8 onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		76: ASM: se.mickelus.tetra.items.modular.impl.toolbelt.booster.TickHandlerBooster@3dbd156 onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
		77: LOWEST
		78: ASM: class com.dragonfight.event.EventHandler onPlayerTick(Lnet/minecraftforge/event/TickEvent$PlayerTickEvent;)V
java.lang.ClassCastException: class net.zepalesque.redux.world.feature.config.TestBelowPlaceConfig cannot be cast to class net.minecraft.world.level.levelgen.feature.configurations.RandomPatchConfiguration (net.zepalesque.redux.world.feature.config.TestBelowPlaceConfig is in module aether_redux@1.19.2 of loader 'TRANSFORMER' @5e1a7d3; net.minecraft.world.level.levelgen.feature.configurations.RandomPatchConfiguration is in module minecraft@1.19.2 of loader 'TRANSFORMER' @5e1a7d3)
	at TRANSFORMER/aether_redux@1.19.2/net.zepalesque.redux.block.natural.ReduxGrassBlock.m_214148_(ReduxGrassBlock.java:58)
	at TRANSFORMER/minecraft@1.19.2/net.minecraft.world.item.BoneMealItem.applyBonemeal(BoneMealItem.java:73)
	at TRANSFORMER/improvableskills@19.2.9/org.zeith.improvableskills.custom.skills.SkillGrowth.growAround(SkillGrowth.java:63)
	at TRANSFORMER/improvableskills@19.2.9/org.zeith.improvableskills.custom.skills.SkillGrowth.tick(SkillGrowth.java:36)
	at TRANSFORMER/improvableskills@19.2.9/org.zeith.improvableskills.api.PlayerSkillData.handleTick(PlayerSkillData.java:77)
	at TRANSFORMER/improvableskills@19.2.9/org.zeith.improvableskills.data.PlayerDataManager.handleDataSafely(PlayerDataManager.java:34)
	at TRANSFORMER/improvableskills@19.2.9/org.zeith.improvableskills.data.PlayerDataManager.playerTick(PlayerDataManager.java:91)
	at TRANSFORMER/improvableskills@19.2.9/org.zeith.improvableskills.data.__PlayerDataManager_playerTick_PlayerTickEvent.invoke(.dynamic)
	at MC-BOOTSTRAP/net.minecraftforge.eventbus/net.minecraftforge.eventbus.ASMEventHandler.invoke(ASMEventHandler.java:73)
	at MC-BOOTSTRAP/net.minecraftforge.eventbus/net.minecraftforge.eventbus.EventBus.post(EventBus.java:315)
	at MC-BOOTSTRAP/net.minecraftforge.eventbus/net.minecraftforge.eventbus.EventBus.post(EventBus.java:296)
	at TRANSFORMER/forge@43.3.5/net.minecraftforge.event.ForgeEventFactory.onPlayerPreTick(ForgeEventFactory.java:814)
	at TRANSFORMER/minecraft@1.19.2/net.minecraft.world.entity.player.Player.m_8119_(Player.java:216)
	at TRANSFORMER/minecraft@1.19.2/net.minecraft.server.level.ServerPlayer.m_9240_(ServerPlayer.java:447)
	at TRANSFORMER/minecraft@1.19.2/net.minecraft.server.network.ServerGamePacketListenerImpl.m_9933_(ServerGamePacketListenerImpl.java:264)
	at TRANSFORMER/minecraft@1.19.2/net.minecraft.network.Connection.m_129483_(Connection.java:248)
	at TRANSFORMER/minecraft@1.19.2/net.minecraft.server.network.ServerConnectionListener.m_9721_(ServerConnectionListener.java:143)
	at TRANSFORMER/minecraft@1.19.2/net.minecraft.server.MinecraftServer.m_5703_(MinecraftServer.java:880)
	at TRANSFORMER/minecraft@1.19.2/net.minecraft.server.dedicated.DedicatedServer.m_5703_(DedicatedServer.java:292)
	at TRANSFORMER/minecraft@1.19.2/net.minecraft.server.MinecraftServer.m_5705_(MinecraftServer.java:806)
	at TRANSFORMER/minecraft@1.19.2/net.minecraft.server.MinecraftServer.m_130011_(MinecraftServer.java:654)
	at TRANSFORMER/minecraft@1.19.2/net.minecraft.server.MinecraftServer.m_206580_(MinecraftServer.java:244)
	at java.base/java.lang.Thread.run(Thread.java:840)
```

### What was the expected result?

Player should not be kicked

### Steps to Reproduce

It currently only happening to one player so far, but we've checked all their mods and they match the server and other players.

### Make sure to confirm these as well:

- [X] The issue persists on the latest version of the mod.
- [X] All version numbers are properly recorded, no instances of ‘latest’
- [X] All details are reported clearly and concisely
- [X] This issue is not a duplicate of another issue
- [ ] If the issue is a modpack-related issue, it has been reported to the modpack devs as well

**Comment by Zepalesque** [comment]
fixed in commit 6c0d92b, once version 2.0 releases this will no longer be an issue

**Comment by Zepalesque** [comment]
fixed in version 2.0
